index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Tribute Page</title>
+
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+
+    <style>
```

Comment:
External styling should be added

Reply:
Same here. Please remember to include why you suggest a change. Some things might be obvious to you but not to someone else. :+1: :smiley: :smiley:

Reply:
- [ ] Please consider editing this comment too, as Santiago suggested! Always ask yourself: How applying the requested changes can improve the code base? In this particular case, How using an external stylesheet instead of using the `<style>` tag can improve the code base?

Please, also consider being more polite when rephrasing your comments, they should sound like suggestions and not orders! 👍🏼 

💡 Hint:

> * This is a polite suggestion: *"I think it would be better to use semantic tags"*
> * This reads as an order: *"Use semantic tags"*

Please, apply these changes! 🤓 🛠️